assets/sass/_variables.scss

```diff
@@ -49,9 +49,9 @@ $form-divider-color: #e8e8e8;
 $facebook-icon-bg-color: #304879; // fb brand color - #3B5998
 $facebook-label-bg-color: #406094;
 $facebook-border-color: #375685;
-$google-icon-bg-color: #d43722; // google brand color - #DC4E41
-$google-label-bg-color: #de4937;
-$google-border-color: #b63c33;
+$google-icon-bg-color: #fff;
+$google-label-bg-color: $button-bg-color;
```

Comment:
imo we should not use the generic variable button background color, since someone that change that variable value to customize widget style doesn't probably want to also affect the Google idp button, which is more an ad hoc style for Google

Reply:
good question. I wouldn't be worry about it for now because
- generic IdP uses it as well
- there is an new design for all IdP buttons (OKTA-199876)